Tests/SwiftyJSONTests/SubscriptTests.swift

```diff
@@ -214,6 +214,48 @@ class SubscriptTests: XCTestCase {
         XCTAssertEqual(json["Type"]["Value"].error, SwiftyJSONError.notExist)
         XCTAssertEqual(json["Type", "Value"].error, SwiftyJSONError.notExist)
     }
+    func testDictionaryDynamicLookupString() {
+        var json: JSON = JSON(rawValue: ["a": "aoo", "bb": "bpp", "z": "zoo"] as NSDictionary)!
+        XCTAssertTrue(json.a == "aoo")
+        XCTAssertEqual(json.bb, JSON("bpp"))
+        XCTAssertTrue(json.z == "zoo")
+        json.bb = "update"
+        XCTAssertTrue(json.a == "aoo")
+        XCTAssertTrue(json.bb == "update")
+        XCTAssertTrue(json.z == "zoo")
+    }
+    func testMultilevelDynamicLookUpGetter() {
+        var json: JSON = ["user": ["id": 987654, "info": ["name": "farshad",
+                                "email": "farshadjahanmanesh@gmail.com"], "feeds": [98833, 23443, 213239, 23232]]]
+        XCTAssertEqual(json["user", "id"], 987654)
```

Comment:
This test case is duplicate? no?

cause the below `testMultilevelDynamicLookUpSetter` both testing setter and getter.

Reply:
sure, done